frontend/src/views/RoleView.vue

```diff
@@ -97,6 +145,77 @@ const createKnowledge = async () => {
 
   knowledge.value.name = ''
 }
+
+const deletedialogVisible = ref(false)
+const levelToDelete = ref(0)
+
+const onClickDeleteLevel = (row: Level) => {
+  levelToDelete.value = Number(row.id)
+  deletedialogVisible.value = true
```

Comment:
```suggestion
  deleteDialogVisible.value = true
```

Respect the law, sir